### ISSUE TITLE
chrome not working 

### ISSUE THREAD

**Issue by koiralabishwas** [linuxmint/mint22-beta#72]
hello there . 
I was trying out linux mint 22 on linve mode . I installed chrome deb package and installed it with gdebi.
I tried to open chrome but it didnot. the mouse cursor spins as smth is loading for a while but nothing happens.
when I run chrome with terminal => google-chrome-stable --no-sandbox       
then only chrome runs.
it maybe the issue with ubuntu 24.04 as I ran in the same issue when I tried it and it also happened here.

I hope this get fixed within the linux mint 22 stable release if possible.

Also I want to know if I am able to upgrade to linux mint 22 from linux mint edge-iso

**Comment by Angedestenebres** [comment]
Hey :cat: 

I tried directly on LM22 already installed on Virtualbox and Chrome works well (by installing the deb from the Google website), see below:

![VirtualBox_Linux Mint 22_03_07_2024_18_22_15](https://github.com/linuxmint/mint22-beta/assets/21110485/c25cca0a-35c7-427e-91c3-646016c7d517)

Maybe you got the issue on liveISO because you didn't do an apt update in the terminal before the installation? Because it seems that the deb file install the package `libu2f-udev` from repository in addition of Chrome browser.

**Comment by koiralabishwas** [comment]
that may be the case .
I will try to launch it after installation .

thank you very much for responding.

**Comment by mtwebster** [comment]
If you continue to have problems, you can you try:

```
apt install apparmor-utils
sudo aa-enforce /etc/apparmor.d/chrome
```

**Comment by koiralabishwas** [comment]
![image](https://github.com/linuxmint/mint22-beta/assets/107906014/cef73d9e-bcb0-4dd2-8fb8-1eb46a3c9f8b)


I installed linux mint 22 in an older hardware and tested it out.
I have that chrome not working problem in live mode , and just after installation.
after I did system update and reboot, chrome worked fine!
but the loading mouse icon apears as it it loading smth for a long , which firefox doesnt.

but looks like it is fine after system upgrade and reboot.

thank you very much for your help
